corehq/apps/cloudcare/middleware.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib import messages
+from django.contrib.auth.models import User
+from django.contrib.auth import login
+
+
+class CloudcareMiddleware(object):
+
+    def process_view(self, request, view_func, view_args, view_kwargs):
+        auth_as = request.GET.get('auth_as', None)
+        view_name = '.'.join((view_func.__module__, view_func.__name__))
```

Comment:
This isn't necessary in the current state, but I was going to use it to check if the view belongs to the cloudcare app.